msteams-platform/resources/teams-updates.md

```diff
@@ -40,21 +40,19 @@ To ensure a smooth transition, a phased rollout of the new platform is planned a
 > [!NOTE]
 >
 > * The new Teams client is supported in Windows and Mac clients and Government Community Cloud (GCC) and GCC-High environments.
-> * We recommend you to test the functionality of  apps, tabs, messaging extensions, bots, and link unfurling on Windows and Mac clients and GCC and GCC-High environments after switching from the Classic Teams client to the new Teams client.
+> * We recommend you to test the functionality of apps, tabs, messaging extensions, bots, and link unfurling on Windows and Mac clients and GCC and GCC-High environments after switching from the Classic Teams client to the new Teams client.
 > * [Adaptive Card tabs](../tabs/how-to/build-adaptive-card-tabs.md) aren't supported in the new Teams client. The Classic Teams client is expected to be deprecated by March 31, 2024. If your app is using Adaptive Card tabs, we recommend you rebuild the tab as a web-based tab. For more information, see [build tabs for Teams](../tabs/what-are-tabs.md).
+> * The `window.alert`, `window.confirm`, and `window.prompt` APIs used to display a dialog aren't supported in the new Teams Client. We recommended you to render a dialog within your own frame, for example, using the [Fluent V9 dialog](https://react.fluentui.dev/?path=/docs/components-dialog--default) or use the Microsoft Teams JavaScript client library (TeamsJS) to display a [Teams dialog](../tabs/what-are-tabs.md) using Adaptive Card or a nested `<iframe>`.
+> * [Location](../concepts/device-capabilities/location-capability.md#location-apis) and [Media](../concepts/device-capabilities/media-capabilities.md#media-capability-apis) APIs aren't supported in the new Teams client. We recommend using HTML5 Geolocation and Media.
```

Comment:
Do we know if these HTML5 equivalents work cleanly across hosts in web/desktop/mobile? Also, we should probably ask platform PM for deliberate short-term/longer-term plan going forward for Location and Media support in new Teams

Reply:
@ydogandjiev @ryanbliss @ChetanSharma-msft

Reply:
It should work across web/desktop/mobile, though cleanly is perhaps subjective. Long term, we think it's worth a discussion on the hub SDK implementing the location APIs without custom hub implementations.

Reply:
It's probably worth including a code snippet of how a dev would request location permissions in a way that works for all platforms. It's likely not elegant, since the way that requesting permissions works will be different across platforms. In the absence of funding for a long-term solution, this should be a pretty good stop gap.